frontend/src/app/support/support.component.css

```
@@ -0,0 +1,764 @@
+  line-height: 1.5;
+  color: inherit;
+  opacity: 0.7;
+  margin-bottom: 1.5rem;
+}
+
+/* Fade-in Animation */
+.fade-in {
+  animation: fadeIn 0.3s cubic-bezier(0.23, 1, 0.32, 1) forwards;
+}
+
+@keyframes fadeIn {
```

Stylelint (17.13.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)